build/ci/jenkins/PR.groovy

```diff
@@ -83,7 +83,7 @@ pipeline {
                     }
                     axis {
                         name 'MILVUS_CLIENT'
-                        values 'pymilvus'
```

Comment:
1. I prefer just run restful testing after orginal testing, so that we can only install milvus for once. Added here will double the resource (including install milvus standalone version, kafka version, pulsar version)
2. Probably add the testing in nightly first,  add it into pr when it is already running stable in nightly.
https://github.com/milvus-io/milvus/blob/master/build/ci/jenkins/Nightly.groovy#L173
@yanliang567  what do you think?

Reply:
updated